### ISSUE TITLE
Loss going to infinity after one minibatch

### ISSUE THREAD

**Issue by rohitgajawada** [meijieru/crnn.pytorch#29]
I made a lmdb dataset from synth90k and the crnn is able to run. But the loss goes to infinity immediately. How can I fix this?
(I also changed imgH to 32)
![screenshot from 2017-06-11 15-40-01](https://user-images.githubusercontent.com/9759196/27010186-44bcad8a-4ebc-11e7-8f97-0b6e596737a1.png)

**Comment by ahmedmazari-dhatim** [comment]
Hi @rohitgajawada , don't worry it's normal. Loss will start decreasing after several iterations . Be patient

**Comment by rohitgajawada** [comment]
I changed the learning rate from 1.0 to 0.0005 and then the loss became normal? Is it supposed to be 1.0?

**Comment by ahmedmazariML** [comment]
@rohitgajawada , let it as it is and wait until the execution finishes

**Comment by rohitgajawada** [comment]
sorry for bothering again but I finished 4 epochs on a dataset of 500k and still network is not learning anything :/ I did not change the learning rate and kept it as 1.0.
![screenshot from 2017-06-13 12-37-50](https://user-images.githubusercontent.com/9759196/27070211-a6ff775a-5035-11e7-9181-1e6d44baac0e.png)

**Comment by ahmedmazari-dhatim** [comment]
@rohitgajawada , don't worry it learns in the followings epochs may be in the 12 epochs or so l don't remember exactly

**Comment by rohitgajawada** [comment]
But shouldn't learning rate be usually small like 0.00005? :P even in the parser arg argument help description its written as :learning rate for Critic, default=0.00005".
I put learning rate as 0.00005 and ran for 2 epochs and I got accuracy of 55% already?

**Comment by ahmedmazari-dhatim** [comment]
@rohitgajawada , you took the full training and validation set ?

**Comment by rohitgajawada** [comment]
training set size is 500k and val size is 50k, my loss also went from around 90 to start with to around 6 in 2 epochs with lr = 0.00005

**Comment by meijieru** [comment]
@rohitgajawada You should adjust the `lr` according to your dataset.

**Comment by ccnankai** [comment]
hi, @rohitgajawada      
the loss   stays around 4  and I have trained two epochs. Has your convergence problem been solved ?
 use create_data.py in lmdb format   ?
@ahmedmazari-dhatim

**Comment by WenmuZhou** [comment]
I meet the same error when training chinese text,I have tried learning rate 0 ,0.1,0.001,0
.0005
